packages/build-utils/package.json

```diff
@@ -23,7 +23,7 @@
     "@types/end-of-stream": "^1.4.0",
     "@types/fs-extra": "9.0.13",
     "@types/glob": "^7.1.1",
-    "@types/jest": "27.0.1",
+    "@types/jest": "27.4.1",
```

Comment:
jest types only at v27 but jest and ts-jest are at 28

Reply:
Unfortunately, there are no `@types/jest` for 28 yet https://www.npmjs.com/package/@types/jest?activeTab=versions

Reply:
I'll take a closer look first thing today to be sure; but if builds pass I assume we are all good too